src/features/metadata-instance-editor/Instance.js

```diff
@@ -209,6 +209,8 @@ class Instance extends React.PureComponent<Props, State> {
             onSave,
         }: Props = this.props;
         const { data: currentData, errors, isCascadingEnabled, isCascadingOverwritten }: State = this.state;
+        const hasCascadePolicy = !!cascadePolicy?.id;
+        const hasCascadePermission = !hasCascadePolicy || (hasCascadePolicy && isCascadingPolicyApplicable);
```

Comment:
I think this boolean logic is the same as:

```
const hasCascadePermission = !cascadePolicy?.id || isCascadingPolicyApplicable;
```

Reply:
This logic has changed